script/Ciel/Day 13/13_01_02_CIEL13_5_2.txt

```diff
@@ -381,27 +436,31 @@ But even after going through all that suffering so she could atone for her sins,
 [sha:82d45c77625db138b546fc41faf2862d54a6e234]{
 -- Page 91, Offset 35520.
 -- 人間として死にたい、だって？
-She wants to die as a human?
+She said she wants to die as a human.
```

Comment:
I think the inquisitive mood is required to link to the above and below.
>Dying as a human? Was that it?

>Because she wants to just forget everything and be at peace?